### PR TITLE
JENKINS-47580 Center align show more button

### PR DESCRIPTION
**Pull request by NicuPascu**
increase css specificity of the show more button so the it center alignes 

See [JENKINS-47580](https://issues.jenkins-ci.org/browse/JENKINS-47580).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-dashboard/src/main/less/core.less [hunk 1]
```diff
@@ -16,7 +16,7 @@
     bottom: auto;
 }
 
-.btn-show-more {
+button.btn-show-more {
     display: table;
```
Comment (sophistifunk) [inline blueocean-dashboard/src/main/less/core.less]: I know you didn't do this @NicuPascu but could you try this with flexbox props on the button? Maybe `align-self:center`? The main container should be `display:flex` already. If it's not, don't worry about it, no need to rejig the whole layout or anything.

**Comment by sophistifunk** [comment]
Just a small question above, but if it's more than a couple of lines to ditch the `display:table` then it's no biggie, I'm just easily offended and should probably HTFU ;-)

🐝

**Comment by NicuPascu** [comment]
I have changed the display: table to flex but can't use `align-self: center` because the container is block, changing that to flex crashed to whole layout so... not sure it is worth it at this moment

**Comment by sophistifunk** [comment]
All good, it's not a big deal anyway. I'll see myself out :)

**Pull request merged**